### PR TITLE
feat(chromium): roll Chromium to M94

### PR DESCRIPTION
**Pull request by aslushnikov**

### CODE REVIEW

**Review by mxschmitt — APPROVED** [review]

File: src/server/chromium/chromium.ts [hunk 1]
```diff
@@ -198,7 +198,7 @@ const DEFAULT_ARGS = [
   '--disable-dev-shm-usage',
   '--disable-extensions',
   // BlinkGenPropertyTrees disabled due to crbug.com/937609
-  '--disable-features=TranslateUI,BlinkGenPropertyTrees,ImprovedCookieControls,SameSiteByDefaultCookies,LazyFrameLoading,GlobalMediaControls,DestroyProfileOnBrowserClose',
+  '--disable-features=TranslateUI,BlinkGenPropertyTrees,ImprovedCookieControls,LazyFrameLoading,GlobalMediaControls,DestroyProfileOnBrowserClose',
```
Comment (dgozman) [inline src/server/chromium/chromium.ts:201]: I cannot find TranslateUI and BlinkGenPropertyTrees features in Chromium source anymore...

File: tests/browsercontext-cookies.spec.ts [hunk 2]
```diff
@@ -137,14 +137,17 @@ it('should get cookies from multiple urls', async ({context}) => {
     url: 'https://foo.com',
     name: 'doggo',
     value: 'woofs',
+    sameSite: 'Lax',
```
Comment (dgozman) [inline tests/browsercontext-cookies.spec.ts:140]: Do I understand correctly that if we add sameSite: None here, we'll get None at line 161? Let's mix and match different SameSite values to see that all of them work?

File: tests/browsercontext-add-cookies.spec.ts [hunk 3]
```diff
@@ -206,7 +206,8 @@ it('should set cookie with reasonable defaults', async ({context, server}) => {
   await context.addCookies([{
     url: server.EMPTY_PAGE,
     name: 'defaults',
-    value: '123456'
+    value: '123456',
+    sameSite: 'Lax',
```
Comment (dgozman) [inline tests/browsercontext-add-cookies.spec.ts:210]: In this particular test we check "reasonable defaults", so I'd rather have different expectations per browser.

File: src/server/chromium/crBrowser.ts [hunk 4]
```diff
@@ -361,7 +361,7 @@ export class CRBrowserContext extends BrowserContext {
   async _doCookies(urls: string[]): Promise<types.NetworkCookie[]> {
     const { cookies } = await this._browser._session.send('Storage.getCookies', { browserContextId: this._browserContextId });
     return network.filterCookies(cookies.map(c => {
-      const copy: any = { sameSite: 'None', ...c };
+      const copy: any = { sameSite: 'Lax', ...c };
```
Comment (dgozman) [inline src/server/chromium/crBrowser.ts:364]: I don't understand this line at all. Do we lack sameSite property sometimes? Why do we assume Lax default now?

Comment (aslushnikov) [reply src/server/chromium/crBrowser.ts:364]: lax is default now in chromium unless specified explicitly. (Used to be 'None')

File: src/server/chromium/chromium.ts (hunk 1 above)
Comment (aslushnikov) [reply src/server/chromium/chromium.ts:201]: cleaned up

File: tests/browsercontext-cookies.spec.ts (hunk 2 above)
Comment (aslushnikov) [reply tests/browsercontext-cookies.spec.ts:140]: done

File: tests/browsercontext-add-cookies.spec.ts (hunk 3 above)
Comment (aslushnikov) [reply tests/browsercontext-add-cookies.spec.ts:210]: Done

**Review by dgozman — APPROVED** [review]

**Pull request merged**